data-provider/class-btm-task-dao.php

```diff
@@ -336,10 +410,21 @@ public function delete_by_id( $id ){
 	public function delete_many_by_ids( array $ids ){
 		global $wpdb;
 
+		if( 0 === count( $ids ) ){
+			return true;
+		}
+
+		$ids_in = '';
+		foreach ( $ids as $id ){
+			$ids_in .= $wpdb->prepare(', %d', $id);
+		}
+
+		$ids_in = ltrim( $ids_in, ', ' );
+
 		$query = $wpdb->prepare('
 			DELETE FROM `' . $this->get_table_name() . '` 
-			WHERE `id` IN ( %s )
-		', implode( ',', $ids ) );
+			WHERE `id` IN ( ' . $ids_in . ' ) AND `status` != "running"
```

Comment:
@eriktad "running" must be defined as constant (at least)

Reply:
I'm forgot fix it

Reply:
Fixed